lib/connectors/kv-memory.js

```diff
@@ -141,3 +159,10 @@ StoreItem.prototype.setTtl = function(ttl) {
     this.expires = undefined;
   }
 };
+
+StoreItem.prototype.getTtl = function() {
+  // Same behaviour as Redis, see http://redis.io/commands/ttl
```

Comment:
IMO, the Redis API is not great from a Node.js/JavaScript perspective.

- For items that don't expire, I expect `ttl()` to return undefined.
- For items that are already expired but were not deleted yet, the same error about non-existent key should be returned.